docs/changelog.md

```diff
@@ -6,22 +6,38 @@
 
 See highlights, bug fixes, and known issues for Streamlit releases:
 
+```eval_rst
+.. tip::
+
+   To upgrade to the latest version of Streamlit, run:
+
+   .. code-block:: bash
+
+      $ pip install --upgrade streamlit
+```
+
 ```eval_rst
 .. contents::
     :local:
     :depth: 1
 ```
 
-```eval_rst
-.. tip::
+## Nightly releases
```

Comment:
I don't think then changelog is the best place for this text. Not least because the features you see in the changelog are not directly related to the nightlies.

How about we create a new page for this instead? It could be called "Try pre-release features". Then this new page would also include explanations about `st.experimental` and `st.beta`, which are coming up soon. So it would look like:

# Try pre-release features

At Streamlit, we like to move fast while keeping things stable. And in our latest effort to move even faster while keeping the promise of stability, we provide our most fearless users with two different ways to try out our most bleeding edge features:

1. Nightly releases
2. Beta and experimental namespaces

## Nightly releases

At the end of each day (at night 🌛), our bots run automated tests against the latest Streamlit code and, if all looks good, publishes them as the `streamlit-nightly` package. This means the nightly build includes our latest features, bug fixes, and other enhancements.

**How this differs from official releases:** official Streamlit releases go through both automated tests and rigorous manual tests, while nightlies only go through automated tests. In addition, new features in nightlies will often lack polish. In our official releases, we make sure all new features are indeed ready for prime time.

**How to use nightlies:** just install the `streamlit-nightly` package:

```
pip uninstall streamlit
pip install streamlit-nightly --upgrade
```

IMPORTANT: You should never have both `streamlit` and `streamlit-nightly` installed in the same environment!

**Why to use nightlies:** because you can't wait for official releases, and/or want to help us find bugs early.

**Why not to use nightlies:** while out automated tests have high coverage, there is still a somewhat high likelihood there will be some bugs lying around. If you do encounter bugs, please report them [here](https://github.com/streamlit/streamlit/issues/new/choose).


## Beta and Experimental Namespaces

[Add test about how st.beta and st.experimental have the latest stuff for you to try. Especially if you couple them with the nightly build. Then we go into more detail about beta/experimental by copy/pasting from [this page](https://www.notion.so/streamlit/Experimental-and-Beta-Namespaces-0781e692281c4359aaf0f9f37fcad58f)]

Reply:
We should also link this page from one of the other installation pages, such as `getting_started.md`

Reply:
@tvst This is a good idea. I would like to understand how we expect users to navigate to this content. Is the expectation that they discover in the left-hand navigation? I still think that we should link to information about Nightly releases from the changelog to ensure that existing users are aware of this. With that said, it doesn't need to be much more than a link -- kind of a "hey if you didn't know about nighlies, now you do".

Let me know what you think. I can get these changes spun up pretty quickly if you agree.

Reply:
I think things look great as-is and would suggest merging this PR.